packages/designto-flutter/painting/painting-box-decoration.ts

```diff
@@ -33,8 +36,25 @@ function fromBackground(b: Background): BoxDecoration {
   }
 }
 
-function fromGradient(): BoxDecoration {
-  throw "not ready.";
+function fromGradient(g: Gradient): BoxDecoration {
+  switch (g._type) {
```

Comment:
will it not be more safer to use `g?._type`

Reply:
fixed!